src/Test.FunctionalTests.Sarif/TestData/Multitool/ValidateCommand/Inputs/SARIF1015.UriMustBeAbsolute_Invalid.sarif

```diff
@@ -42,21 +42,7 @@
           ]
         }
       ],
-      "columnKind": "utf16CodeUnits",
-      "properties": {
-        "expectedResults": {
```

Comment:
> "expectedResults": { [](start = 7, length = 21)

thanks for explanation offline, i remember using these in tests a long whiel ago. Glad to hear we no longer have a need for these.